Dockerfile

```diff
@@ -0,0 +1,20 @@
+# #################################### BUILD STAGE ####################################
+FROM maven as build-phase
+COPY . /app
+WORKDIR /app
+RUN mvn dependency:resolve
+RUN mvn clean package
+# RUN mvn clean package -DskipTests
+
+# #################################### RUN STAGE ####################################
+FROM jitsi/jicofo:latest
+
+COPY --from=build-phase /app /app
+
+WORKDIR /
+
+RUN cp /app/target/jicofo-1.1-SNAPSHOT-jar-with-dependencies.jar /usr/share/jicofo/jicofo.jar
```

Comment:
What will happen if the jar name changed?..Please rename the jar file or mentioned it as * for eg: jicofo*.jar